tasks/create_record.json

```diff
@@ -1,16 +1,16 @@
 {
     "puppet_task_version": 1,
     "supports_noop": false,
-    "description": "Create a ServiceNow object",
+    "description": "Create a ServiceNow record\nbolt task run --nodes dev84270 servicenow_tasks::create_record table=incident fields='{\"description\":\"An example incident.\"}'",
```

Comment:
What does `bolt task show` output for this task? What does `bolt task show <task>` look like? Curious to see if the multi-line description does anything weird in the output.

Reply:
I messed around with a few different ways to format it, and this one isn't great, but it's better than having it all on one line.

Reply:
Yeah, looks like `bolt task show <task_name>` outputs a stub usage. So we probably don't need the `bolt task run` part.